parts/linux/cloud-init/artifacts/components.json

```diff
@@ -40,14 +40,21 @@
       "downloadURL": "mcr.microsoft.com/containernetworking/azure-cni:*",
       "amd64OnlyVersions": [],
       "multiArchVersions": [
-        "v1.5.28"
+        "v1.5.28",
+        "v1.6.0"
```

Comment:
Do we need to cache this new version on Windows VHDs?

Reply:
Yes, 1.6.0 is only going to be applied on K8s 1.30 and version 1.5.X will still remains active for K8s < 1.30